packages/block-editor/src/components/media-placeholder/README.md

```diff
@@ -85,7 +85,15 @@ If false the default placeholder style is used.
 - Type: `Boolean`
 - Required: No
 - Default: `false`
-- Platform: Web
+- Platform: Web | Mobile
+
+### disableMediaSelection
```

Comment:
We have a button to select from the media library, a button to upload a new media file and sometimes a button to insert from a URL. I guess given that the property name is disableMediaSelection users may think it only disables the media selection button while leaving the upload button.
I'm not sure of what a good name would be, maybe `disableMediaUI`, `disableMediaButtons`.